Sources/Playbook/Components/User/Multiple Users Indicator/PBMultipleUsersIndicator.swift

```diff
@@ -17,7 +17,7 @@ public struct PBMultipleUsersIndicator: View {
     if let count = usersCount, count != 0 {
       Text("+\(count)")
         .tag("additionalUser")
-        .pbFont(.buttonText(size.fontSize), color: .pbPrimary)
+        .pbFont(size == .xxSmall ?.buttonText(size.fontSize) : .badgeText, color: .pbPrimary)
```

Comment:
Let's turn this "size == .xxSmall ?.buttonText(size.fontSize) : .badgeText, color: .pbPrimary" into a computed property.  and pass the prop to .pbFont modifier instead of calculating the font value in run time.